assets/styles/authoring/community.scss

```diff
@@ -0,0 +1,20 @@
+#communities-table .pagination .page-item {
+  &.active {
```

Comment:
These style definitions/overrides for the paging component should be removed.  I'm not sure what the intent was here, but these are causing the "Paging" component to render (in light mode) with dark gray backgrounds that make it difficult to read the text.  Removing lines 1-15 seems to restore the correct rendering.

Reply:
We changed the color of the pagination component from green to some kind of gray due to some suggestion we received. In order not to contaminate other parts of the app, we only did it for communities. Maybe we can try other colors to make it more visible in light mode or if we prefer consistency, we can remove it without problem

Reply:
@mcass19 IMO probably just revert color change, and we can address later as part of upcoming design work.